### ISSUE TITLE
how can i load a model in my pc`s jupyter notebook trained in colab using weight files

### ISSUE THREAD

**Issue by rexayyy** [meituan/YOLOv6#407]

**Comment by mtjhl** [comment]
You can set the pretrained model path here:
https://github.com/meituan/YOLOv6/blob/ab76a39b9f89c15424ad6366794306239caaf573/configs/yolov6s.py#L4

**Comment by rexayyy** [comment]
i have trained a model in pytorch script using yolov6 in colab. I have the weight files with me. How can i load my model trained on custom data.

**Comment by mtjhl** [comment]
Same answer.

> You can set the pretrained model path here:
> 
> https://github.com/meituan/YOLOv6/blob/ab76a39b9f89c15424ad6366794306239caaf573/configs/yolov6s.py#L4

**Comment by rexayyy** [comment]
No, i am asking you about the a custom trained model not a pretrained model. like you load one using model.load command
